src/components/AddBookForm.js

```diff
@@ -165,14 +199,55 @@ const Form = styled.form`
   }
 
   button {
-    background-color: #504465;
+    background-color: #006a75;
     display: flex;
     justify-content: center;
     align-items: center;
     height: 2.5rem;
   }
 `
 
+const Wrapper = styled.div`
+  background-color: #fff;
+  border-radius: 25px;
+  box-shadow: var(--box-shadow);
+  padding: 1rem 1rem 1.5rem 1rem;
```

Comment:
```suggestion
  padding: 1rem 1rem 1.5rem;
```
You dont need the last one.